crates/red_knot_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -237,14 +272,19 @@ use super::constraint::Constraint;
 mod bitset;
 mod symbol_state;
 
+pub(crate) type AllConstraints<'db> = IndexVec<ScopedConstraintId, Constraint<'db>>;
```

Comment:
I think this type no longer needs to be `pub(crate)`, it's only used in this file.